public/jira/derived/derive.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  DerivedWorkTiming,
+  deriveWorkTiming,
+  WorkTimingConfig,
+} from "./work-timing/work-timing.js";
+import {
+  DerivedWorkStatus,
+  getWorkStatus,
+  WorkStatusConfig,
+} from "./work-status/work-status.js";
+import {
+  normalizeIssue,
+  NormalizeIssueConfig,
+} from "../normalized/normalize.js";
+import { JiraIssue, NormalizedIssue } from "../shared/types.js";
+
+/**
+ * @typedef {import("../shared/types.js").NormalizedIssue & {
+ *   derivedTiming: import("./work-timing/work-timing.js").DerivedWorkTiming
+ * } & {derivedStatus: import("./work-status/work-status.ts").DerivedWorkStatus}} DerivedIssue
+ */
+export type DerivedIssue = NormalizedIssue & {
+  derivedTiming: DerivedWorkTiming;
+  derivedStatus: DerivedWorkStatus;
+};
+
+/**
+ * Adds derived data
+ * @param {NormalizedIssue} normalizedIssue
+ * @return {DerivedIssue}
+ */
+export function deriveIssue(
+  issue: NormalizedIssue,
+  options: Partial<WorkStatusConfig & WorkTimingConfig> & {
+    uncertaintyWeight?: number;
+  } = {}
+): DerivedIssue {
+  const derivedTiming = deriveWorkTiming(issue, options);
```

Comment:
`options` isn't being used, given the pattern in the other files I think these (`deriveWorkTiming` and `getWorkStatus`) should be defaulted on options

Reply:
`options` is intended for the callbacks being used under `deriveWorkTiming` and `getWorkStatus` where it gets a default value there.

Its not meant for passing an implementation to `deriveWorkTiming` and `getWorkStatus`.

Does that make sense?